shopinvader_api_delivery_carrier/services/delivery.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.component.core import Component
+
+
+class DeliveryService(Component):
+    """Service for getting information on stock.picking"""
+
+    _inherit = ["base.shopinvader.service", "abstract.shopinvader.download"]
+    _name = "shopinvader.delivery.service"
+    _usage = "deliveries"
+    _expose_model = "stock.picking"
+    _description = __doc__
+
+    def _get_report_action(self, target, params=None):
+        return self.env.ref("stock.action_report_delivery").report_action(
+            target, config=False
+        )
```

Comment:
This must still be migrated to a FastAPI service.
Waiting for https://github.com/shopinvader/odoo-shopinvader/pull/1466 to progress, and see if the helper addon is created in OCA.

Reply:
We don't add it for now as we don't need it, but it can be added later on.